src/software/test_util/equal_within_tolerance.h

```diff
@@ -14,6 +16,21 @@
 
 namespace TestUtil
 {
+    /**
+     * Checks if two durations are within tolerance of each other
+     * Two durations are within tolerance of each other if they are within tolerance of
+     * each other
```

Comment:
:joy: this was really funny you got an audible laugh out of me, can probably keep one or the other

Reply:
ah I meant to update this